plugins/pulp_ostree/plugins/lib.py

```diff
@@ -360,9 +394,15 @@ def list_refs(self):
         _list = []
         lib = Lib()
         self.open()
-        flags = lib.OSTree.RepoPullFlags.COMMIT_ONLY
-        _, summary = self.impl.remote_list_refs(self.id, None)
+        try:
+            _, summary = self.impl.remote_list_refs(self.id, None)
+        except lib.GLib.GError:
+            if not required:
+                summary = {}
+            else:
+                raise
```

Comment:
The docblock says `LibError` will be raised, but I think this will raise `lib.GLib.GError`